packages/back-end/src/models/DimensionMetadataModel.ts

```diff
@@ -0,0 +1,119 @@
+import mongoose from "mongoose";
+import uniqid from "uniqid";
+import { omit } from "lodash";
+import { DimensionMetadataInterface } from "../types/Integration";
+import { queriesSchema } from "./QueryModel";
+
+const dimensionMetadatachema = new mongoose.Schema({
```

Comment:
I'd probably change the name of this model to be less abstract (metadata schema) to more specific (dimension value queries, etc)